src/components/Dialog/index.jsx

```diff
@@ -0,0 +1,168 @@
+import ListItem from "@material-ui/core/ListItem";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import Dialog from "@material-ui/core/Dialog";
+import Visibility from "@material-ui/icons/Visibility";
+import VisibilityOff from "@material-ui/icons/VisibilityOff";
+import InputAdornment from "@material-ui/core/InputAdornment";
+import IconButton from "@material-ui/core/IconButton";
+import Input from "@material-ui/core/Input";
+
+const useStyles = makeStyles(theme => ({
+  textField: {
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(1),
+    width: 200
+  },
+  btnPrimary: {
+    marginBottom: 100,
+    marginTop: 50,
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "center"
+  },
+  btnBottom: {
+    display: "flex",
+    justifyContent: "space-around"
+  }
+}));
+
+function SimpleDialog(props) {
+  const classes = useStyles();
+  const { onClose, open } = props;
+  const [values, setValues] = React.useState({
+    password: "",
+    showPassword: false
+  });
+
+  const handleChange = prop => event => {
+    setValues({ ...values, [prop]: event.target.value });
+  };
+
+  const handleClickShowPassword = () => {
+    setValues({ ...values, showPassword: !values.showPassword });
+  };
+
+  const handleMouseDownPassword = event => {
+    event.preventDefault();
+  };
+
+  const handleListItemClick = value => {
+    onClose(value);
+  };
+
+  return (
+    <Dialog aria-labelledby="dialog-title" open={open}>
+      <DialogTitle id="dialog-title" className="dialog">
+        Welcome Back We've Missed You!
+      </DialogTitle>
+      <List>
+        <ListItem>
+          <TextField
+            className={classes.textField}
+            id="margin-none"
+            label="Email/Username"
+          />
+        </ListItem>
+        <ListItem>
+          <Input
+            id="margin-none"
+            placeholder="Password"
+            className={classes.textField}
+            type={values.showPassword ? "text" : "password"}
+            value={values.password}
+            onChange={handleChange("password")}
+            endAdornment={
+              <InputAdornment position="end">
+                <IconButton
+                  aria-label="toggle password visibility"
+                  onClick={handleClickShowPassword}
+                  onMouseDown={handleMouseDownPassword}
+                >
+                  {values.showPassword ? <Visibility /> : <VisibilityOff />}
+                </IconButton>
+              </InputAdornment>
+            }
+          />
+        </ListItem>
+        <ListItem>
+          <Button
+            variant="outlined"
+            onClick={() => handleListItemClick()}
+            color="primary"
+          >
+            LOG IN
+          </Button>
+        </ListItem>
+      </List>
+    </Dialog>
+  );
+}
+
+SimpleDialog.propTypes = {
+  onClose: PropTypes.func.isRequired,
+  open: PropTypes.bool.isRequired
+};
+
+export default function DialogInput() {
```

Comment:
put this component into a different file, React shouldn't have multiple components in the same file, unless they are really small, also why is it called `DialogInput` from components content it seems more like `About` component